userguide/install_kernel_modules.rst

```diff
@@ -0,0 +1,71 @@
+Install Kernel Modules
+======================
+
+Install with DKMS
+^^^^^^^^^^^^^^^^^
+
+In order to support the mandatory kernel subsystems ashmem and binder for the
+Android container you have to install two
+`DKMS <https://en.wikipedia.org/wiki/Dynamic_Kernel_Module_Support>`_
+based kernel modules. The source for the kernel modules is maintained by the
+Anbox project `here <https://github.com/anbox/anbox-modules>`_.
+
+At the moment we only have packages prepared for Ubuntu in a PPA on Launchpad.
+If you want to help to get the packages in your favorite distribution please
+come and talk to us or submit a PR with the distribution specific packaging.
+
+Install DKMS package from PPA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to add the PPA to your Ubuntu system please run the following commands:
+
+.. code-block:: text
+
+    $ sudo add-apt-repository ppa:morphis/anbox-support
+    $ sudo apt update
+    $ sudo apt install linux-headers-generic anbox-modules-dkms
+
+These will add the PPA to your system and install the `anbox-modules-dkms`
+package which contains the ashmem and binder kernel modules. They will be
+automatically rebuild everytime the kernel packages on your system update.
+
+After you installed the `anbox-modules-dkms` package you have to manually
+load the kernel modules. The next time your system starts they will be
+automatically loaded.
```

Comment:
On a side note, is it easily possible to automatically load the kernel modules within the package installation so that the modprobe steps below are not necessary?